internal/server/proto/server.proto

```diff
@@ -1426,6 +1433,25 @@ message Job {
     InitResult init = 14;
   }
 
+  message Config {
+    // Source is the location where the configuration was loaded from.
+    Source source = 1;
+
+    // Source is the location where the configuration was loaded from.
+    enum Source {
```

Comment:
Should this be a `oneof` in case we ever support any other source? 🤔

Reply:
We can add fields to the enum! The only reason to do `oneof` is if we want to attach data to the sources. I thought about that but wasn't sure since today both would probably be `google.protobuf.Empty` and I couldn't think of anything here.

Reply:
Ah, roger that. Makes sense 👍🏻